bedrock/externalpages/templates/externalpages/pocket/privacy.html

```diff
@@ -0,0 +1,142 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+{{ css_bundle('pocket-privacy') }}
+{% endblock %}
+
+{% block page_title %}Privacy{% endblock page_title %}
+
+{% block content %}
+  {% include 'externalpages/pocket/includes/nav.html' %}
+  <div class="mzp-l-content mzp-t-content-xl pocket-privacy-wrapper">
+    <section>
+      <h1>Privacy Policy</h1>
+      <h2 class="date-heading">Posted October 11, 2021</h2>
+      <p>This notice describes the Privacy Policy that applies to all information collected or submitted when you install or use the Pocket application or any of the other proprietary software applications or services we provide, and when you visit our website (all of which are referred to collectively herein as the "Pocket Technologies"). The Pocket Technologies are owned and licensed by Mozilla Corporation (referred to herein as "we," "us," "our," "Mozilla"), and are operated by Mozilla's business unit known as "Pocket," which has developed applications and services for use in connection with a variety of devices and platforms, including without limitation, Windows, Mac, iPhone, iPad, Kindle Fire, Android, BlackBerry, Windows phone, Chrome, Safari, Firefox and other mobile devices and computers.</p>
+      <p>We understand your concerns about privacy. We want you to understand what types of information we collect and what may happen to that information if you install the Pocket application, visit the Pocket Website or use any of the Pocket Technologies.</p>
+      <p>Our Privacy Policy includes the following provisions, and you can jump to each by selecting the links below:</p>
+      <ul>
+        <li><a href="#user">User Accounts and Collection of Information</a></li>
+        <li><a href="#collection">Collection of Information as Part of Using Pocket Service</a></li>
+        <li><a href="#cookies">Use of Cookies and Analytics</a></li>
+        <li><a href="#info">Use of Information</a></li>
+        <li><a href="#sharing">Sharing of Information</a></li>
+        <li><a href="#keeping">Keeping Information</a></li>
+        <li><a href="#transfer">Transfer of Information</a></li>
+        <li><a href="#keeping">Keeping Information</a></li>
+        <li><a href="#security">Security</a></li>
+        <li><a href="">Phishing</a></li>
+        <li><a href="">Managing My Privacy and Email Settings</a></li>
+        <li><a href="">Amendments to Policy</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+    </section>
+
+    <section id="user">
+      <h2>USER ACCOUNTS AND COLLECTION OF INFORMATION</h2>
```

Comment:
I highly recommend that you should make words uppercase via CSS and not through HTML as it can make readability harder especially for folks using screen readers, as screen readers can sometimes read letter-by-letter if all text is written in uppercase in HTML files.

```
//css
      h2 {
      text-transform: uppercase
      }
```

Reply:
💯